www_src/components/option-panel/option-panel.jsx

```diff
@@ -0,0 +1,82 @@
+var React = require('react/addons');
+
+// Internal -- just the container
+var OptionPanel = React.createClass({
+  render: function () {
+    return (<div className="option-panel">
+      {this.props.children}
+    </div>);
+  }
+});
+
+var Checkbox = React.createClass({
+  mixins: [React.addons.LinkedStateMixin],
+  getDefaultProps: function () {
+    return {
+      linkState: this.linkState,
+      truthy: true,
+      falsey: false
+    };
+  },
+  onChange: function (e) {
+    var val = e.target.checked ? this.props.truthy : this.props.falsey;
+    this.valueLink.requestChange(val);
+  },
+  isTruthy: function () {
+    return this.valueLink.value === this.props.truthy;
+  },
+  render: function () {
+    var linkState = this.props.linkState;
+    this.valueLink = linkState(this.props.id);
+    return (<label className={'label ' + (this.isTruthy() ? 'selected' : '')}>
```

Comment:
if you return `false` instead of `''`, className won't even be bound as property (so you don't end up with a `class=""` in the final DOM